client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -54,7 +54,20 @@ export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) =>
 		// If the user's email is unverified, we still want to show the trial plan option
 		migrationTrialEligibility?.error_code === 'email-unverified';
 
+	const plans = Plans.useSitePlans( { siteId: site.ID } );
+	const pricing = plans.data ? plans.data[ visiblePlan ].pricing : undefined;
```

Comment:
Non-blocking, but it may be worth using a conditional accessor here, just in case `plans.data[ visiblePlan ]` isn't present for some reason. It just means we'll handle the situation smoothly rather than triggering a nasty error.
```suggestion
	const pricing = plans.data ? plans.data[ visiblePlan ]?.pricing : undefined;
```